src/Workflow/Schema/SubmitAction.php

```diff
@@ -44,12 +44,12 @@ public function __construct(
         private string $actionType,
         #[Property(description: 'id of the element', type: 'integer', example: 50)]
         private int $elementId,
-        #[Property(description: 'type of the element', type: 'string', example: ElementTypes::TYPE_OBJECT)]
+        #[Property(description: 'type of the element', type: 'string', example: ElementTypes::TYPE_DATA_OBJECT)]
         private string $elementType,
-        #[Property(description: 'name of the workflow', type: 'string', example: 'my_first_workflow')]
-        private string $workflowName,
-        #[Property(description: 'transition', type: 'string', example: 'start_workflow')]
-        private string $transition,
+        #[Property(description: 'id of the workflow', type: 'string', example: 'my_first_workflow')]
+        private string $workflowId,
+        #[Property(description: 'id of the transition', type: 'string', example: 'start_workflow')]
```

Comment:
```suggestion
        #[Property(description: 'ID of the transition', type: 'string', example: 'start_workflow')]
```

Reply:
Afaik we use id in the descirption of params ...
e.g
![image](https://github.com/user-attachments/assets/d155e1ef-7a23-4901-a74f-409b8023e085)